### DISCUSSION TITLE
브라우저 확장 프로그램을 10개월동안 운영하면서 느낀 것들 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#16] [Blog Comments]
# 브라우저 확장 프로그램을 10개월동안 운영하면서 느낀 것들 - peterkimzz

올해 초 가상화폐 투자가 유행하면서 코인 시세 1초만에 보는 크롬 확장 프로그램 만들기 라는 글을 작성했고, 가상화폐 관련 확장 프로그램을 크롬 웹 스토어에 게시했습니다.
중간에 다른 사이드 프로젝트를 하느라 업데이트를 2-3개월 정도 안하긴 했지만, 꾸준하게 사용해주

https://www.peterkimzz.com/briana-extension-review/

**Comment by jshong225** [comment]
어쩌다 들어왔는데 글 이쁘게 정리 잘되어있어 재밌게 읽었습니다. 감사합니다😀

**Reply by peterkimzz** [reply]
재밌게 읽어주셔서 감사합니다 ^^ 종종 비슷한 주제로 글 작성하겠습니다!
